include/linear_feedback_controller_msgs/eigen_conversions.hpp

```diff
@@ -40,16 +39,52 @@ struct Control {
 };
 }  // namespace Eigen
 
+// Create an alias to use only one namespace here.
+inline void wrenchEigenToMsg(const ::Eigen::Matrix<double, 6, 1>& e,
+                             geometry_msgs::msg::Wrench& m) {
+  m.force.x = e(0);
+  m.force.y = e(1);
+  m.force.z = e(2);
+  m.torque.x = e(3);
+  m.torque.y = e(4);
+  m.torque.z = e(5);
```

Comment:
```suggestion
  tf2::toMsg(e.template head<3>(), m.force);
  tf2::toMsg(e.template tail<3>(), m.torque);
```
In case tf2_eigen stays there